apps/backend/src/app/api/latest/team-invitations/accept/verification-code-handler.tsx

```
@@ -67,7 +69,7 @@ export const teamInvitationCodeHandler = createVerificationCodeHandler({
 
     return codeObj;
   },
-  async handler(tenancy, {}, data, body, user) {
+  async handler(tenancy, { }, data, body, user) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)